variables.tf

```diff
@@ -0,0 +1,132 @@
+// Required
+variable "name" {
+  default     = ""
+  description = "Elastic Search Service cluster name."
+  type        = "string"
+}
+
+variable "subnet_ids" {
+  description = "List of VPC Subnet IDs for the Elastic Search Service EndPoints will be created."
+  type = "list"
+}
+
+variable "vpc_id" {
+  default     = ""
+  description = "Vpc id where the Elastic Search Service cluster will be launched."
+  type        = "string"
+}
+
+variable "zone_id" {
+  default     = "Route 53 zone id where our "
+  description = ""
+  type        = "string"
+}
+
+// Optional
+variable "access_policies" {
+  default     = ""
+  description = "IAM policy document specifying the access policies for the domain."
+  type        = "string"
+}
+
+variable "dedicated_master" {
+  default     = false
+  description = "Indicates whether our cluster have dedicated master nodes enabled."
+  type        = "string"
+}
+
+variable "encryption_enabled" {
+  default     = "no"
```

Comment:
default value here need to be "false", "no" its not valid according this:
https://www.terraform.io/docs/providers/aws/r/elasticsearch_domain.html#enabled

Reply:
Sorry... I created this in a initial state and then changed its values but seems that I forgot to change this.

P.D - > Gracias majo!